resin/README.md

```diff
@@ -0,0 +1,47 @@
+# Agent Check: Resin
+
+## Overview
+
+This check monitors [Resin][1] through the Datadog Agent.
+
+## Setup
+
+### Installation
+
+The Resin check is not included in the [Datadog Agent][2] package, so you will
+need to install it yourself.
+
+### Configuration
+
+1. Edit the `resin.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your resin performance data. See the [sample resin.d/conf.yaml][2] for all available configuration options.
+
+2. [Restart the Agent][3].
+
+### Validation
+
+[Run the Agent's status subcommand][4] and look for `resin` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][6] for a list of metrics provided by this integration.
+
+### Service Checks
+
+Resin instance is running and reporting JVM metrics
```

Comment:
```suggestion
**resin.can_connect**:

Returns `CRITICAL` if the Agent is unable to connect to and collect metrics from the monitored Resin instance. Returns `OK` otherwise.
```